playwright.config.ts

```diff
@@ -3,19 +3,30 @@ import { configDotenv } from "dotenv";
 
 configDotenv();
 
+const reporters: any[] = [
+  ["list"],
+  ["rollwright/coverage-reporter", { name: "text" }],
+  ["rollwright/coverage-reporter", { name: "lcovonly", options: { file: "lcov-functional.info" } }],
+];
+
+if (process.env.CI) {
+  const vscodeVersion = process.env.VSCODE_VERSION ?? "stable";
+  reporters.push([
+    "junit",
+    {
+      outputFile: "TEST-result-webview.xml",
+      includeProjectInTestName: true,
+      suiteName: `VS Code (${vscodeVersion}) Extension Tests: Webview (${process.platform} ${process.arch})`,
```

Comment:
I'm not entirely convinced this line is used like we'd expect since `semaphore.mk` has to handle its own naming when publishing test result files